src/parser/jobs/mnk/modules/CelestialRevolution.tsx

```diff
@@ -0,0 +1,42 @@
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+export class CelestialRevolution extends Analyser {
+	static override handle = 'celestialRevolution'
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private count: number = 0
+
+	override initialise(): void {
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.type('action')
+				.action(this.data.actions.CELESTIAL_REVOLUTION.id), () => this.count++)
+		this.addEventHook('complete', this.onComplete)
```

Comment:
Style nit, if you're gonna multi-line then multi-line the whole thing. The callback arrow function can be it's own line. Also prefer a newline between multi-line hooks and complete so it's easier to read. Hopefully GH doesn't mangle whitespace on this suggestion.
```suggestion
		this.addEventHook(
			filter<Event>()
				.source(this.parser.actor.id)
				.type('action')
				.action(this.data.actions.CELESTIAL_REVOLUTION.id),
			() => this.count++
		)

		this.addEventHook('complete', this.onComplete)
```